prow/jobs.md

```diff
@@ -244,5 +264,18 @@ Batch Job:
 
 See ["How to test a ProwJob"](/prow/build_test_update.md#How-to-test-a-ProwJob).
 
-[`Presets`]: https://github.com/kubernetes/test-infra/blob/3afb608d28630b99e49e09dd101a96c201268739/prow/config/jobs.go#L33-L40
+## Badges
+
+Prow can display badges that signal whether jobs are passing ([example](https://prow.k8s.io/badge.svg?jobs=post-test-infra-bazel)).
+
+The format to send your `deck` URL is `/badge.svg?jobs=single-job-name` or `/badge.svg?jobs=common-job-prefix-*`.
+
```

Comment:
Concrete example?

Reply:
Isn't example in the previous paragraph concrete?

Reply:
...Yup it is! ><